inject/src/main/java/io/micronaut/context/DefaultBeanContext.java

```diff
@@ -1797,7 +1797,7 @@ public <T> Optional<T> findBean(@Nullable BeanResolutionContext resolutionContex
             }
         } catch (DisabledBeanException e) {
             if (AbstractBeanContextConditional.ConditionLog.LOG.isDebugEnabled()) {
-                AbstractBeanContextConditional.ConditionLog.LOG.debug("Bean of type [{}] disabled for reason: {}", beanType.getSimpleName(), e.getMessage());
+                AbstractBeanContextConditional.ConditionLog.LOG.debug("Bean of type [{}] disabled for reason: {}", beanType.getSimpleName(), e.getMessage(), e);
```

Comment:
Stacktrace unlikely to add value here.
```suggestion
                AbstractBeanContextConditional.ConditionLog.LOG.debug("Bean of type [{}] disabled for reason: {}", beanType.getSimpleName(), e.getMessage());
```